puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
The only problem with that is when the job fails or is aborted, and something goes wrong with the cleanup (e.g. abort was pressed twice), the gemset will never get cleaned up and we may slowly run out of disk space on slaves.  If there's a chance the gemset will get reused then it might get cleaned up properly later.

The reason I've been thinking of using just build executor is that we might be able to remove the cleanup and leave gems installed, making subsequent builds faster (Gemfile.lock gets recreated, but the gem is already installed).

I think using just build executor without any job name would be fine, irrespective of removing cleanup or not.